boot/zephyr/Kconfig

```diff
@@ -388,6 +388,15 @@ config RECOVERY_UART_DEV_NAME
 
 endif # MCUBOOT_SERIAL
 
+config BOOT_INTR_VEC_RELOC
+	bool "Relocate the interrupt vector to the application"
+	default n
+	depends on SW_VECTOR_RELAY || CPU_CORTEX_M_HAS_VTOR
```

Comment:
Does it make sense for this feature to be 'n' if either of these are set? I'm wondering if this should be a Kconfig, or just a define in the code based on these other defines.